vpr/src/base/read_options.cpp

```diff
@@ -1705,6 +1710,11 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("-2")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_timing_grp.add_argument<bool, ParseOnOff>(args.disable_check_rr_graph, "--disable_check_rr_graph")
+        .help("Disables checking rr graph when reading from disk.")
+        .default_value("off")
+        .show_in(argparse::ShowIn::HELP_ONLY);
```

Comment:
I'd probably invert this to be an option `--check_rr_graph` which defaults to `on` and applies whenever an RR graph is created (e.g. algorithmically or loaded from disk). That seems more intuitive and easier to explain. End users who don't want to pay the check time when loading from disk can then disable it on the command line.

Reply:
Sure, done